manifests/yum.pp

```diff
@@ -37,28 +69,24 @@
 #     ``$servers`` entries appropriately
 #
 # @param simp_gpg_url
-#   A specially crafted string that handles GPG url creation
+#   A specially crafted string that handles GPG URL creation
 #
 #   * The string ``YUM_SERVER`` (all caps) will be replaced with the various
 #     ``$servers`` entries appropriately
 #
 class simp::yum (
-  Simplib::Netlist                                     $servers,
-  Boolean                                              $enable_simp_repos   = true,
-  Boolean                                              $enable_os_repos     = true,
-  Boolean                                              $enable_auto_updates = true,
-  Variant[Stdlib::HTTPSUrl, Stdlib::HTTPUrl]           $os_update_url       = "https://YUM_SERVER/yum/${facts['os']['name']}/${facts['os']['release']['major']}/${facts['architecture']}/Updates",
-  Variant[Stdlib::HTTPSUrl, Stdlib::HTTPUrl, Enum['']] $os_gpg_url          = '',
-  Variant[Stdlib::HTTPSUrl, Stdlib::HTTPUrl]           $simp_update_url     = "https://YUM_SERVER/yum/SIMP/${facts['architecture']}",
-  Variant[Stdlib::HTTPSUrl, Stdlib::HTTPUrl, Enum['']] $simp_gpg_url        = ''
+  Boolean                                              $enable_auto_updates        = true,
+  Boolean                                              $enable_simp_local_repos    = false,
```

Comment:
Because $enable_simp_local_repos defaults to false, this code is NOT backward compatible with previous release.  As such, you should bump the major version number.